MySimpleSiteStatistics/README.md

```diff
@@ -1,4 +1,67 @@
 # My Simple Site Statistics
 ([Italian translate](README_IT.md))
 
-[TO TRANSLATE]
+## Index
+
+- [Preface](#preface)
+- [Instructions](#instructions)
+- [Clone the repository](#clone_the_repository)  
+- [Features to implement](#features-to-implement)    
+        - [Add more view of data visualization [database]](#add_more_view_of_data_visualization)    
+        - [Add others stored procedure of insertion [database, unit test]](#add_stored_procedure_of_insertion)   
+  
+## Preface
+
+Mark is developping a system for registrartion of the accesses to websites.  
+To implement that he created several tables to store data, some stored procedure to insert them and the views for facilitate of visualization.  
+
+Jacob, a colleague of Tom, who is interested to the project, asked a for a series of features for adding to the project to implement it in its own blog and finally having a way to find out how many accesses receives its own site.  
+
+Mark who given the unexpected feature request, needs help. Jacob suggested your name and now Mark asked you to implement some of the features.    
```

Comment:
maybe: "Mark given the unexpected feature request"

I mean, Mark read the mail of Tom and say: What a lot of feature! I cannot to do all alone! (something like this :D )

Reply:
ok. for this sentence: "Jacob suggested your name ..." is it ok? I was wondered about this one, but probably because I just translated and didn't do the project yet and I did not understand it is. Thank you :)

Reply:
I think yes. I mean that one people speak to another about his skill that it's the right people for this job... so Jim (Giacomo) has speaked to Mark about you (who make learn by doing) and Mark take you for make the tasks :) It's "passaparola" :) the most useful system for recruiting people ;)

Reply:
ok, so i have to inform about that system, I will be useful ;) thank you .